ext/standard/tests/streams/bug78326.phpt

```diff
@@ -0,0 +1,16 @@
+--TEST--
+memory allocation on stream_get_contents()
+--FILE--
+<?php
+$f = tmpfile();
+fwrite($f, '.');
+
+$chunks = array();
+for ($i = 0; $i < 1000; ++$i) {
+    rewind($f);
+    $chunks[] = stream_get_contents($f, 1000000);
+}
+var_dump(count($chunks));
+?>
+--EXPECT--
+int(1000)
```

Comment:
Should probably have an explicit memory limit configured?

Reply:
It's quite weird to have an environment with 1GB (the total memory the test would allocate without the patch). But yeah, it would not fail.

Reply:
My local environment usually has the memory limit disabled on CLI.